test/integration-legacy/runner-spec.test.js

```diff
@@ -251,7 +251,7 @@ describe('Runner', function () {
                             expect(cursor.ref).to.eql(runStore.ref);
 
                             expect(response.code).to.be(200);
-                            expect(reason.lookup(response.code).name).to.be(response.status);
+                            expect(response.status).to.be(reason.lookup(response.code).name);
```

Comment:
This is not a valid test. You're looking up a system that itself is generating the assertion!!!

Match the status and detail with manual text.